### PR TITLE
Fix: Run php-cs-fixer

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] runs `php-cs-fixer`

💁‍♂️ Should https://fabbot.io be enabled for this repository?

### CODE REVIEW

File: tests/Configurator/CopyDirectoryFromPackageConfiguratorTest.php [hunk 1]
```diff
@@ -115,17 +114,18 @@ private function cleanUpTargetFiles()
...
+    private function rrmdir($dir)
+    {
         if (is_dir($dir)) {
             $objects = scandir($dir);
             foreach ($objects as $object) {
-                if ($object != "." && $object != "..") {
-                    if (filetype($dir."/".$object) == "dir") {
-                        $this->rrmdir($dir."/".$object);
+                if ('.' != $object && '..' != $object) {
+                    if ('dir' == filetype($dir.'/'.$object)) {
```
Comment (Pierstoval) [inline tests/Configurator/CopyDirectoryFromPackageConfiguratorTest.php]: Isn't it possible to use `===` in these statements instead?

Comment (localheinz) [reply tests/Configurator/CopyDirectoryFromPackageConfiguratorTest.php]: Sure, but out of scope of this PR. This PR is only about applying changes from running `php-cs-fixer`.

**Comment by fabpot** [comment]
Thank you @localheinz.

**Comment by fabpot** [comment]
I've also enabled fabbot.io for the repository

**Comment by localheinz** [comment]
Thank you, @fabpot and @Pierstoval!

**Pull request merged**